sdk/src/main/scala/coop/rchain/sdk/dag/merging/DagMergingLogic.scala

```diff
@@ -86,15 +85,6 @@ object DagMergingLogic {
     partitionScope(sorted)
   }
 
-  /** Lowest fringe across number of fringes. */
-  def lowestFringe[B: Ordering](fringes: Set[Set[B]], height: B => Long): Set[B] = {
```

Comment:
Because of three reasons:
1. It doesn't work correctly with `Set(Set())`
1. It's confusing that is referring to fringe and height although B is abstract type
1. Requires ordering on B which should not be relevant.

Fringe related calculations should be part of the finalizer as it already calculates latest fringe.